packages/fluentui/docs/src/examples/components/Dropdown/State/DropdownExampleLoading.shorthand.tsx

```diff
@@ -15,6 +15,21 @@ const DropdownExampleLoading = () => {
       search
       items={inputItems}
       placeholder="Start typing a name"
+      getA11yStatusMessage={({ resultCount, previousResultCount }) => {
+        if (loading) {
```

Comment:
do we want to have the props in this example?
if yes should be better to add to this example as well following ?:
` const [loading] = useBooleanKnob({ name: 'loading', initialValue: true });`

Reply:
Are you sure? I am hitting that line every time when loading is true.

<img width="1043" alt="Screenshot 2020-04-28 at 9 12 36 AM" src="https://user-images.githubusercontent.com/11275392/80453208-7b66ed80-8930-11ea-800d-0f27b6bb6208.png">

Reply:
sorry I see I commented wrong file, I wanted to comment the file below
DropdownExampleHeaderMessage.shorthand.tsx 

yes in this file it was working fine :)